driver-async/src/test/functional/com/mongodb/async/client/AbstractUnifiedTest.java

```diff
@@ -84,7 +80,7 @@
 
 // See https://github.com/mongodb/specifications/tree/master/source/transactions/tests
 @RunWith(Parameterized.class)
-public class TransactionsTest {
+public abstract class AbstractUnifiedTest {
```

Comment:
Since localizing the operations in the subclasses seems awkward, I decided to get a head start on unifying the tests. I have renamed the class to reflect this.